OrbitGl/App.cpp

```diff
@@ -671,7 +691,7 @@ outcome::result<void, std::string> OrbitApp::OnLoadCapture(
   ar.time_graph_ = GCurrentTimeGraph;
   OUTCOME_TRY(ar.Load(file_name));
 
-  m_ModulesDataView->SetProcess(Capture::GTargetProcess);
+  //  m_ModulesDataView->SetProcess(Capture::GTargetProcess);
```

Comment:
Why is this kept as a comment?

Reply:
I was wondering if it breaks something - I have not checked actually thanks for noticing, I will check save and load.